### DISCUSSION TITLE
Animated hitcircleoverlay is broken?

### DISCUSSION THREAD

**Discussion by taptapking** [ppy/osu#17427] [Q&A] [answered]
I don't know why it doesn't work anymore after the latest update. Is this a measure to stop those EZ exploit skins?
![screenshot1244](https://user-images.githubusercontent.com/34684004/159862843-1a30aa56-1af2-4d20-b0bc-5f92be6bc3fd.png)

**Comment by peppy** [comment, marked as answer]
This repository is for lazer discussions.

If you are going to report an issue, please at bare minimum include a link to the skin or beatmap affected

https://github.com/ppy/osu-stable-issues/issues is the place you can report issues.
